tools/fileinfo/features/rich-header/test.py

```diff
@@ -0,0 +1,22 @@
+from regression_tests import *
+
+class Test_001(Test):
+    settings=TestSettings(
+        tool='fileinfo',
+        args='--json --verbose',
+        input=['file-32bit.ex_', 'file-64bit.ex_']
+    )
+
+    def test_rich_header_values(self):
```

Comment:
This test and the two added files to `tools/fileinfo/features/rich-header` (basically the whole commit 80678a1d0761fe6524f2d2589f5a4b88ae62ebb7) is already in `master` (added in #41). I suggest to base the branch on the `current` master instead of on commit 80678a1d0761fe6524f2d2589f5a4b88ae62ebb7. You can do that in the following way:
```
# Update master to the current version on GitHub.
git checkout master
git pull

# Get back to the branch.
git checkout LZ_DetectZeroEntryPoint

# Base the branch on the current master.
git rebase master

# Push the changed branch (--force is needed because we have rewritten history).
git push --force
```